CMakeLists.txt

```diff
@@ -124,6 +126,8 @@ if (WIN32)
     target_link_libraries(sycl_blas PRIVATE ComputeCpp::ComputeCpp)
   elseif(is_dpcpp)
     target_link_libraries(sycl_blas PRIVATE DPCPP::DPCPP)
+  elseif(is_hipsycl)
+    target_link_libraries(sycl_blas PRIVATE hipSYCL::hipSYCL-rt)
```

Comment:
I think it'd be worth splitting out hipSYCL support and most other CMake changes (without FPGA support) into a separate PR, I believe we should be able to get that in much faster.